app/om/src/main/java/uk/ac/ebi/fg/annotare2/submission/model/FileType.java

```diff
@@ -21,11 +21,11 @@
+    RAW_FILE("Raw", "The \"native\" files generated by the microarray scanner software (e.g. .CEL, .gpr) or sequencing machine (.fastq.gz) <br/><a href=\"https://www.ebi.ac.uk/fg/annotare/help/accepted_raw_ma_file_formats.html\" target=\"_blank\"><i class=\"fa fa-life-ring\" aria-hidden=\"true\"></i>Accepted microarray raw file formats</a><br/><a href=\"https://www.ebi.ac.uk/fg/annotare/help/accepted_sequencing_file_formats.html#raw_data_files\" target=\"_blank\"><i class=\"fa fa-life-ring\" aria-hidden=\"true\"></i>Accepted sequencing raw file formats</a> <div> <img src=\"../../assets/images/graphic_raw.png\" alt=\"One Color\" width=\"250\" height=\"125\"></img> </div>"),
 
     RAW_MATRIX_FILE("Raw Matrix", "A raw file in tab-delimited text (.txt) format, that contains data from more than one assay (probes/genes in rows, assays in columns) <br/><a href=\"https://www.ebi.ac.uk/fg/annotare/help/strict_mtab_matrix.html\" target=\"_blank\"><i class=\"fa fa-life-ring\" aria-hidden=\"true\"></i>Accepted matrix file formats</a><div> <img src=\"../../assets/images/graphic_raw_matrix.png\" alt=\"One Color\" width=\"175\" height=\"125\"></img> </div>"),
 
-    PROCESSED_FILE("Processed", "A tab-delimited text (.txt) file containing data from a single assay <br/><a href=\"https://www.ebi.ac.uk/fg/annotare/help/accepted_processed_ma_file_formats.html\" target=\"_blank\"><i class=\"fa fa-life-ring\" aria-hidden=\"true\"></i>Accepted microarray processed file formats</a><br/><a href=\"https://www.ebi.ac.uk/arrayexpress/help/UHTS_submissions.html#processed_data\" target=\"_blank\"><i class=\"fa fa-life-ring\" aria-hidden=\"true\"></i>Accepted sequencing processed file formats</a><div> <img src=\"../../assets/images/graphic_processed.png\" alt=\"One Color\" width=\"250\" height=\"125\"></img> </div> "),
+    PROCESSED_FILE("Processed", "A tab-delimited text (.txt) file containing data from a single assay <br/><a href=\"https://www.ebi.ac.uk/fg/annotare/help/accepted_processed_ma_file_formats.html\" target=\"_blank\"><i class=\"fa fa-life-ring\" aria-hidden=\"true\"></i>Accepted microarray processed file formats</a><br/><a href=\"https://wwwdev.ebi.ac.uk/fg/annotare/help/accepted_sequencing_file_formats.html#processed_data\" target=\"_blank\"><i class=\"fa fa-life-ring\" aria-hidden=\"true\"></i>Accepted sequencing processed file formats</a><div> <img src=\"../../assets/images/graphic_processed.png\" alt=\"One Color\" width=\"250\" height=\"125\"></img> </div> "),
```

Comment:
Can we please change this link from pointing to `wwwdev` to `www`

Reply:
Good catch. Copy/paste error...
Can we use relative links from this page? Or is it better to keep the full URL?

Reply:
We can use relative links as well :) Usually using relative links is generally considered best practice :)